### DISCUSSION TITLE
Helm Chart not using the right Docker images tags

### DISCUSSION THREAD

**Discussion by ogui11aume** [Budibase/budibase#11223] [Help] [answered]
Hi
I noticed yesterday when using Helm Chart according to doc here https://docs.budibase.com/docs/kubernetes-k8s#install-budibase-helm-chart 
that the images being pulled having tags like 2.8.10 however, the image should actually be pulled have tags : **v**2.8.10 with the **v** before the version number. 

Example : 
Helm chart references budibase/apps:2.8.10
Actual image is budibase/apps:**v**2.8.10

This is true for the other project specific images.

There is the problem with the tag names used in the helm chart.
Change in tag naming at some point ?

Actual image on Docker Hub
https://hub.docker.com/layers/budibase/apps/v2.8.10/images/sha256-abfd79a8947a7e2bb45e01d1db8dec01126a3e383bd9277e2024380b67feffaf?context=explore

**Comment by melohagan** [comment, marked as answer]
Hey @ogui11aume 

There's an open issue you can follow here: https://github.com/Budibase/budibase/issues/10543

We are looking to fix this one soon!
